saifa/docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: "3.7"
+
+services:
+  backend:
+    image: patranakamo/saifa-be:v1.3.3
```

Comment:
For each image in this compose file, we need to add the multi-arch digest to make sure that we are locking in a specific image that has been tested for this release. With just the tag, a new image can be pushed to docker hub with the same tag and then a new image would be downloaded by the user just by restarting the app (and of course by fresh installs).

Your image is already multi-arch which is fantastic:
<img width="498" alt="image" src="https://github.com/getumbrel/umbrel-apps/assets/85373263/4d8e0b37-20d4-4871-aae4-30349f1a2600">

So all you need to do is get the digest by running:
```
docker pull patranakamo/saifa-be:v1.3.3
```

and grabbing the digest from the output. In this case I get:
```
Digest: sha256:a04bbaba219b35d7bb59426665ae813ad2d219cb1b2e0eceed1c79170ede6228
```

So in this example you would do:
```
image: patranakamo/saifa-be:v1.3.3@sha256:a04bbaba219b35d7bb59426665ae813ad2d219cb1b2e0eceed1c79170ede6228
```